packages/template-retail-react-app/app/utils/product-utils.test.js

```diff
@@ -49,3 +53,10 @@ test('getDisplayVariationValues', () => {
         Width: 'M'
     })
 })
+
+test('getDisplayPrice', () => {
```

Comment:
Add an additional test for when there's no discounted price.

Reply:
added.